Microsoft.Azure.CosmosRepository/src/Dtos/IPagedResultRequest.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Microsoft.Azure.CosmosRepository.Dtos
+{
+    /// <summary>
+    /// This interface is defined to standardize to request a paged result.
+    /// </summary>
+    public interface IPagedResultRequest : ILimitedResultRequest
+    {
+        /// <summary>
+        /// Skip count (beginning of the page).
+        /// </summary>
+        int SkipCount { get; set; }
```

Comment:
```suggestion
        int SkipCount { get; }
```

Hide set to consumers, the implementation can still assign to it - but we do not want consumers setting this.

Reply:
Actually, the consumers will set the property when they want to pagination, below is the demo code,  maybe I am wrong.
```
var pagedResultRequest = new PagedResultRequest { SkipCount = input.SkipCount, MaxResultCount = input.MaxResultCount };
var mediaList = await _mediaRepository.GetListAsync(predicate, pagedResultRequest);
```